relocated lis to dp section/add redirect (#1457)
cp-algorithms/cp-algorithms: src/dynamic_programming/intro-to-dp.md

## Classic Dynamic Programming Problems
| Name                                           | Description/Example                                                                                                                                                                                                            |
| ---------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | [0-1 Knapsack](https://cp-algorithms.com/dynamic_programming/knapsack.html)                                   | Given $W$, $N$, and $N$ items with weights $w_i$ and values $v_i$, what is the maximum $\sum_{i=1}^{k} v_i$ for each subset of items of size $k$ ($1 \le k \le N$) while ensuring $\sum_{i=1}^{k} w_i \le W$?                  |

Reviewer: ```suggestion
| [0-1 Knapsack](../dynamic_programming/knapsack.md)                                   | Given $W$, $N$, and $N$ items with weights $w_i$ and values $v_i$, what is the maximum $\sum_{i=1}^{k} v_i$ for each subset of items of size $k$ ($1 \le k \le N$) while ensuring $\sum_{i=1}^{k} w_i \le W$?                  |
```

| Subset Sum                                     | Given $N$ integers and $T$, determine whether there exists a subset of the given set whose elements sum up to the $T$.                                                                                                         |

> | [Longest Increasing Subsequence (LIS)](https://cp-algorithms.com/dynamic_programming/longest_increasing_subsequence.html)           | You are given an array containing $N$ integers. Your task is to determine the LIS in the array, i.e., a subsequence where every element is larger than the previous one.                                                       |

Reviewer: ```suggestion
| [Longest Increasing Subsequence (LIS)](../dynamic_programming/longest_increasing_subsequence.md)           | You are given an array containing $N$ integers. Your task is to determine the LIS in the array, i.e., a subsequence where every element is larger than the previous one.                                                       |
```

| Edit Distance                                  | The edit distance between two strings is the minimum number of operations required to transform one string into the other. Operations are ["Add", "Remove", "Replace"]                                                         |

## Related Topics

> * [Bitmask Dynamic Programming](https://cp-algorithms.com/dynamic_programming/profile-dynamics.html)

Reviewer: ```suggestion
* [Bitmask Dynamic Programming](../dynamic_programming/profile-dynamics.md)
```
